src/views/sources/viewSourcesList.tsx

```diff
@@ -336,8 +336,10 @@ const SourcesListView: React.FunctionComponent = () => {
                 icon={<EmptyStateIcon icon={PlusCircleIcon} />}
               />
               <EmptyStateBody>
-                Begin by adding a source. A source contains a collection of network information, including systems
-                management solution information, IP addresses, or host names, in addition to SSH ports and credentials.
+                To add a source, ensure that the necessary credentials have been provided. A source contains key system
+                information such as system management details, IP addresses, hostnames, and connection configurations,
+                including SSL settings and options for tools like Paramiko or OpenSSH. The credentials you previously
+                added will be used to authenticate and scan the source.
```

Comment:
optional/future:
move strings into locale file since we already handle strings in this component